src/search.c

```diff
@@ -1627,38 +1626,18 @@ on_find_in_files_dialog_response(GtkDialog *dialog, gint response,
 
 
 static gboolean
-search_find_in_files(const gchar *utf8_search_text, const gchar *dir, const gchar *opts,
+search_find_in_files(const gchar *utf8_search_text, const gchar *utf8_dir, const gchar *cmd,
 	const gchar *enc)
 {
-	gchar **argv_prefix, **argv, **opts_argv;
-	gchar *command_grep;
+	gchar **argv = g_new(gchar *, 4);
```

Comment:
if the size is know and the memory freed unconditionally on function exist, maybe using a stack-allocated array would be just as fine, wouldn't it?